packages/cubejs-duckdb-driver/src/DuckDBDriver.ts

```diff
@@ -26,10 +26,14 @@ type InitPromise = {
 export class DuckDBDriver extends BaseDriver implements DriverInterface {
   protected initPromise: Promise<InitPromise> | null = null;
 
+  private schema: string;
+
   public constructor(
     protected readonly config: DuckDBDriverConfiguration = {},
   ) {
     super();
+
+    this.schema = getEnv('duckdbSchema', this.config);
```

Comment:
Let's allow the override schema by passing it with config. We use the same approach for all drivers.